### PR TITLE
Updated readme.md

### PR DESCRIPTION
**Pull request by Twaha-Rahman**
Updated the README.md file with all the required info to get started.

This PR should close both #1 and #3

### CODE REVIEW

**Comment by Twaha-Rahman** [comment]
This PR should also close #8 . As I have added a `Disclaimer` too

**Review by bradtaniguchi — CHANGES_REQUESTED** [review]
We **may** want to end up changing where the files are saved to another branch in `git` using [this approach](https://medium.com/@minamimunakata/how-to-store-images-for-use-in-readme-md-on-github-9fb54256e951). This way when someone downloads the repo they don't have to download all the extra png/gif files.

I'd say the easiest way to integrate that approach at this point is to **remove the image files, but leave the links in the README as is**, and I'll add a branch following the above approach and add the images and "fix" the links.
(I already downloaded the images you used locally)

So just remove/update the disclaimer as a PR has already added this, but leave the rest of the docs as they look pretty good. @nhcarrigan Would you like to review the updates to the README to check for typos, any syntax issues (I only quickly skimmed em ;D)

**Comment by naomi-lgbt** [comment]
I have to step out for a bit, but I see a couple of typos and some grammar changes. I'll add some comments when I get back.

**Comment by Twaha-Rahman** [comment]
> I'd say the easiest way to integrate that approach at this point is to **remove the image files, but leave the links in the README as is**, and I'll add a branch following the above approach and add the images and "fix" the links.

I removed **all** the images along with the folder

> So just remove/update the disclaimer as a PR has already added this, but leave the rest......

Reverted and removed the disclaimer

All set for a review! 😃

File: README.md [hunk 1]
```diff
@@ -1,23 +1,85 @@
...
+So, you want to contribute to this repo? We are glad to recieve PRs from anybody who is interested. Follow these steps if you are new to contributing to open source projects -
 
-- [Nodejs@12](https://nodejs.org/en/blog/release/v12.13.0/)
-- [nvm](https://github.com/nvm-sh/nvm)
-- [nvm-windows](https://github.com/coreybutler/nvm-windows)
+1. [Install the required softwares](#install-prerequisites)
+2. Fork the repo
+   ![GIF - How to fork on GitHub](./README-images/fork-resized.gif)
+3. Clone your forked repo
+4. Create a new branch on the cloned repo and swith to it
```
Comment (naomi-lgbt) [inline README.md]: Typo: swith => switch

Comment (Twaha-Rahman) [reply README.md]: Fixed it!

**Comment by naomi-lgbt** [comment]
@bradtaniguchi You're good to merge this... there's a couple of grammar pieces I want to tweak but it'd be easier to do it as a new pull request I think

**Comment by Ryuno-Ki** [comment]
@nhcarrigan Are you going to fix the conflicting file?

**Comment by bradtaniguchi** [comment]
I was waiting for @Twaha-Rahman to fix the conflicts (as its their PR) before merging. Otherwise I'll resolve it myself later today and merge it.

**Comment by Twaha-Rahman** [comment]
@bradtaniguchi Sorry I couldn't reply earlier....was doing other stuffs... removed the merge conflict, good to go!

**Comment by bradtaniguchi** [comment]
> @bradtaniguchi Sorry I couldn't reply earlier....was doing other stuffs... removed the merge conflict, good to go!

No worries! No rush at all :D

**Pull request merged**